classes/order/OrderSlip.php

```diff
@@ -307,7 +307,7 @@ public static function create(Order $order, $product_list, $shipping_cost = fals
         $order_slip->partial = 0;
 
         if ($shipping_cost !== false) {
-            $order_slip->shipping_cost = true;
+            $order_slip->shipping_cost = 1;
```

Comment:
Because in `classes/order/OrderSlip.php`, `shipping_cost` is an integer

Reply:
Then I think it should be fixed, this is clearly a boolean it's even one of the rare that was correctly used in the legacy classes (unlike other fields configured as boolean which uses integer values)

So let's not follow blindly (stupidly) our static analysis tool by following the incorrect type that was set, we should instead interpret the inconsistency that was raised by the tool and update the configuration to TYPE_BOOL, it will have no impact on the DB anyway.

Reply:
Fixed